StardewArchipelago/Locations/CodeInjections/Vanilla/RecipeLevelUpInjections.cs

```diff
@@ -71,15 +71,13 @@ private static void SendSkillCraftingRecipeChecks(Skill skill, int level)
         }
 
         // public LevelUpMenu(string skillName, int level)
-        /*public static void SkillLevelUpMenuConstructor_SendModdedSkillRecipeChecks_Postfix(IClickableMenu __instance, string skillName, int level)
+        public static void SkillLevelUpMenuConstructor_SendModdedSkillRecipeChecks_Postfix(IClickableMenu __instance, string skillName, int level)

```

Comment:
This method seems to be missing a verification that the skill is valid. I think it'll hook into random unsupported modded skills, and other similar stuff

Reply:
I believe I resolved this but double check if its what you were wanting.